src/test/java/com/onarandombox/multiverseinventories/TestWorldChanged.java

```diff
@@ -353,6 +353,170 @@ public void testGroupedSharesWorldChange() throws Exception {
         Assert.assertEquals(originalInventory, newInventory);
     }
 
+    @Test
+    public void testLastLocation() throws Exception {
+
+        // Initialize a fake command
+        Command mockCommand = mock(Command.class);
+        when(mockCommand.getName()).thenReturn("mvinv");
```

Comment:
Looking at this from a reviewer's perspective makes me realize that this could easily be streamlined into some kind of helper/util methods.

Reply:
I think it would be better to save that for another PR.

Reply:
Of course.